cmd/neofs-cli/modules/session/create.go

```diff
@@ -68,18 +74,21 @@ func createSession(cmd *cobra.Command, _ []string) {
 	endpoint, _ := cmd.Flags().GetString(commonflags.RPC)
 	currEpoch, err := internalclient.GetCurrentEpoch(ctx, endpoint)
 	common.ExitOnErr(cmd, "can't get current epoch: %w", err)
+	cmd.Println("Current epoch:", currEpoch)
 
 	exp, _ := cmd.Flags().GetUint64(commonflags.ExpireAt)
 	lifetime := uint64(defaultLifetime)
-	if lfArg, _ := cmd.Flags().GetUint64(commonflags.Lifetime); lfArg != 0 {
-		exp = currEpoch + lfArg
+	if cmd.Flags().Changed(commonflags.Lifetime) {
```

Comment:
IINW may be shortened to
```go
	var exp uint64
	if cmd.Flags().Changed(commonflags.ExpireAt) {
		// Cobra guarantees that commonflags.Lifetime is unset then (see init)
		exp, _ = cmd.Flags().GetUint64(commonflags.ExpireAt)
	} else {
		lifetime, _ := cmd.Flags().GetUint64(commonflags.Lifetime)
		exp = currEpoch + lifetime
	}
	if exp <= currEpoch {
		common.ExitOnErr(cmd, "", errors.New("expiration epoch must be greater than current epoch"))
	}
```

Reply:
I think we also need a check of passing --lifetime 0

Reply:
suggest to add separate check for this

Reply:
technically enough, but smth like `lifetime must not be zero` would be more clearer, right?